src/ui/oled/oled.c

```diff
@@ -257,6 +257,7 @@ void oled_mirror(bool mirror)
     } else {
         _write_cmd(OLED_CMD_SET_SEGMENT_RE_MAP_COL127_SEG0);
         _write_cmd(OLED_CMD_SET_COM_OUTPUT_SCAN_UP);
+        _write_cmd_with_param(OLED_CMD_SET_DISPLAY_OFFSET, DISPLAY_OFFSET);
```

Comment:
Makes sense. This is also what's set in the `oled_init` at the beginning.
While there, I would also make a `#define` for the other offset of `0x20` when mirrored.

Reply:
Done